uisdk/src/main/res/drawable/kh_uisdk_capacities_circle.xml

```diff
@@ -4,7 +4,7 @@
     android:shape="oval">
 
     <solid
-        android:color="@color/kh_uisdk_capabilities_number_background"/>
+        android:color="@color/kh_uisdk_background_primary"/>
```

Comment:
May need to remove this background_primary from the color list and use one of the other background colors